docs/CLI.md

```diff
@@ -318,6 +318,24 @@ If configuration files are found in the specified paths, _all_ projects specifie
 
 :::
 
+### `--randomize`
+
+Shuffle the order of the tests within a file. The seed used to generate the new order will be printed to stdout.
+
+:::tip
+Additionally you can use the [`--seed`](#--seed) flag to pass the seed value:
+
+    ```bash
+    jest --randomize --seed 1234
+    ```
```

Comment:
Please remove these spaces in the beginnings of lines. I can’t submit a suggestion, it does not work with code block (or at least I don’t know how to make it work).

Reply:
Thanks. Looks good.